schema/new_library_verification_cases.schema.json

```diff
@@ -0,0 +1,1062 @@
+{
```

Comment:
I think that we might want to change the name of that file to something more generic such as `verification_cases.schema.json`, I always though that the `new_library_` prefix was a bit confusing since it's not the same thing at all as the `library.json` file.